MSMAll/scripts/SingleSubjectConcat.sh

```diff
@@ -31,504 +31,171 @@
 #
 #~ND~END~
 
```

Comment:
I prefer to have `set -eu` on new-format scripts, which can catch when you make a typo on a variable name, or use a variable before you set it.  The only false positives I know of are if you are used to the lazy "default empty string" behavior (I think it is better style to explicitly do `var=""`, as it makes it obvious what behavior you wanted), or intentionally wanting an empty array to expand to zero words (fairly rare, has been fixed in new bash versions, and there is a workaround for old bash).

We have workarounds to deal with sourcing external things like fix's `settings.sh`, which by default isn't clean for exit codes and undefined variables, but since you don't source anything external here, you shouldn't need to use them.